src/components/TaskDetails/TaskDetailsStyle.scss

```diff
@@ -0,0 +1,32 @@
+.MainModalTD{
```

Comment:
it's common to use camelCase for naming variables, but kebab-case for css classes 

camelCase
snake_case
kebab-case

FYI: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849